src/guide/installation.md

```diff
@@ -47,112 +39,112 @@ $ npm install vue@next
 
 ## CLI
 
-Vue provides an [official CLI](https://github.com/vuejs/vue-cli) for quickly scaffolding ambitious Single Page Applications. It provides batteries-included build setups for a modern frontend workflow. It takes only a few minutes to get up and running with hot-reload, lint-on-save, and production-ready builds. See [the Vue CLI docs](https://cli.vuejs.org) for more details.
+Vue provee una [interfaz de línea de comandos (CLI) oficial](https://github.com/vuejs/vue-cli) para realizar la creación rápida de la estructura base para las Single Page Applications. Esta provee _setups_ con baterías incluidas para un flujo de trabajo de frontend moderno. Solo toma unos minutos configurar y ejecutarlo con _hot-reload_, _lint-on-save_, y compilaciones listas para producción. Véase [the Vue CLI docs](https://cli.vuejs.org) para más detalles.
```

Comment:
Maybe we could write something like this. @Tintef @sxcamacho @migsalazar What do you guys think?

```suggestion
Vue provee una [interfaz de línea de comandos (CLI) oficial](https://github.com/vuejs/vue-cli) para realizar la creación rápida de la estructura base para las Single Page Applications. Proporciona configuraciones de _build_ con baterías incluidas para un proceso de trabajo de _front-end_ moderno. Solo toma unos minutos configurar y ejecutarlo con _hot-reload_, _lint-on-save_, y compilaciones listas para producción. Véase [the Vue CLI docs](https://cli.vuejs.org) para más detalles.
```

Reply:
The current docs are using "prontas", so maybe that's the way.

But, IMHO, "prontas" isn't clear for me (topic-related: spanish in different countries). If I may make a suggestion, maybe something like this could work:

```suggestion
Vue provee una [interfaz de línea de comandos (CLI) oficial](https://github.com/vuejs/vue-cli) para realizar la creación rápida de la estructura base para las Single Page Applications. Proporciona configuraciones de _build_ listas para su uso en un proceso de trabajo de _front-end_ moderno. Solo toma unos minutos configurar y ejecutarlo con _hot-reload_, _lint-on-save_, y compilaciones listas para producción. Véase [the Vue CLI docs](https://cli.vuejs.org) para más detalles.
```

Reply:
I like the one @migsalazar proposed the most. I'm not a big fan of using `baterías`, it sounds weird in Spanish IMO

Reply:
Yes, @migsalazar got a better one!